Marlin/pins_PRINTRBOARD_REVF.h

```diff
@@ -117,17 +154,54 @@
 // Temperature Sensors
 //
 #define TEMP_0_PIN          1   // Analog Input (Extruder)
+#define TEMP_A_PIN          2   // Analog Input (Extrudrboard A THERM)
+#define TEMP_B_PIN          3   // Analog Input (Extrudrboard B THERM)
 #define TEMP_BED_PIN        0   // Analog Input (Bed)
 
 //
 // Heaters / Fans
 //
 #define HEATER_0_PIN       15   // C5 PWM3B - Extruder
-#define HEATER_1_PIN       44   // F6
-#define HEATER_2_PIN       45   // F7
+// Note that Printrboard REV F6 removed HEATER_A_PIN from EXP1 and
+// reassigned it for a second fan output.
+#define HEATER_A_PIN       44   // F6 - Extrudrboard A HOTEND
+#define HEATER_B_PIN       45   // F7 - Extrudrboard B HOTEND
 #define HEATER_BED_PIN     14   // C4 PWM3C
 
 #define FAN_PIN            16   // C6 PWM3A
+#define PRINTRBOARD_F6_HEATER_FAN_PIN 44 // pin F6 on REV F6, see above.
+
+#ifndef NO_EXTRUDRBOARD
+#ifndef NO_EXTRUDRBOARD_OUTPUT_SWAP
+
+  #define E1_STEP_PIN        EA_STEP_PIN
+  #define E1_DIR_PIN         EA_DIR_PIN
+  #define E1_ENABLE_PIN      EA_ENABLE_PIN
+  #define TEMP_1_PIN         TEMP_A_PIN
+  #define HEATER_1_PIN       HEATER_A_PIN
+
+  #define E2_STEP_PIN        EB_STEP_PIN
+  #define E2_DIR_PIN         EB_DIR_PIN
+  #define E2_ENABLE_PIN      EB_ENABLE_PIN
+  #define TEMP_2_PIN         TEMP_B_PIN
+  #define HEATER_2_PIN       HEATER_B_PIN
+
+#else
+
+  #define E1_STEP_PIN        EB_STEP_PIN
+  #define E1_DIR_PIN         EB_DIR_PIN
+  #define E1_ENABLE_PIN      EB_ENABLE_PIN
+  #define TEMP_1_PIN         TEMP_B_PIN
+  #define HEATER_1_PIN       HEATER_B_PIN
+
+  #define E2_STEP_PIN        EA_STEP_PIN
+  #define E2_DIR_PIN         EA_DIR_PIN
+  #define E2_ENABLE_PIN      EA_ENABLE_PIN
+  #define TEMP_2_PIN         TEMP_A_PIN
+  #define HEATER_2_PIN       HEATER_A_PIN
+
+#endif // NO_EXTRUDRBOARD_OUTPUT_SWAP
```

Comment:
I get the general reasoning for doing it with intermediate named pins, such as for easier re-assignment, but really this is not needed. The pins file, once set, doesn't change, and if users need to edit the pins file for reassignment I'm sure they can figure out which of these blocks applies.